docs/sources/explore/explore-inspector.md

```diff
@@ -1,97 +1,87 @@
+- **Error tab -** Shows any errors. _Only visible if the query returns an error._
 
-### Open the Inspector
+## Query inspector Stats tab
 
-1. Run the query you would like to inspect.
-1. Click the **Inspector** button.
+You can inspect query performance in the **Stats tab**, which displays statistics that tell you how long your query takes, how many queries you send, the number of rows returned and trace IDs. This information can help you troubleshoot your queries, especially if any of the numbers are unexpectedly high or low.
+
+1. Open the inspector.
+1. Click the **Stats tab**.
+
+Statistics display in read-only format.
 
-The inspector pane opens on the bottom of the screen.
+## Query inspector Query tab
 
-### Inspect raw query results
+View raw request and response in the Query tab.
 
-You can view raw query results, that is the data returned by the query in a table.
+1. Open the Query inspector and click the **Query tab**.
+1. Click **Refresh**.
 
-In the **Inspector** tab, click the **Data** tab.
+Grafana sends the query to the server and displays the result. You can drill down on specific portions of the query, expand or collapse all of it. Click **Copy to clipboard** to copy the data to use in other applications.
 
-For multiple queries or for queries multiple nodes, there are additional options.
+## Query inspector JSON tab
 
-- **Show data frame:** Select the result set data you want to view.
-- **Series joined by time:** View the raw data from all of your queries at once, one result set per column. You can click a column heading to sort the data.
+View data results as JSON and as data frame JSON models in the **JSON tab**.
 
-### Download raw query results as CSV
+1. Open the Query inspector and click the **JSON tab**.
+1. Choose one of the following options from the **Select source** dropdown menu:
+   - **Panel data -** Displays a JSON object representing the data retrieved from Explore.
+   - **DataFrame JSON (from query) -** Displays the raw data result set without transformations and field configuration applied.
 
-Grafana generates a CSV file in your default browser download location. You can open it in the viewer of your choice.
+## Query inspector Data tab
 
-1. In the **Inspector** tab, get raw query results by following the instructions [described in Inspect raw query results](#inspect-raw-query-results).
-1. Refine query settings until you can see the raw data that you want to export.
-1. Click **Download CSV**.
+View, inspect and download raw query results in the **Data tab**.
 
-In order to download a CSV file specifically formatted for Excel, expand **Data options** and then enable the **Download for Excel** toggle before you click the **Download CSV** option.
+1. Open the Query inspector and click the **Data** tab.
+1. Click **Data options** to to view options under **Show data frame**.
+1. Select a data results set from the dropdown menu.
```

Comment:
The dropdown is shown only when there are multiple series returned:

<img width="855" alt="Screenshot 2024-07-10 at 11 47 02" src="https://github.com/grafana/grafana/assets/745532/873b9dab-4c72-43cc-a16a-5175df11a057">


<img width="650" alt="Screenshot 2024-07-10 at 11 46 27" src="https://github.com/grafana/grafana/assets/745532/b41082f8-087f-4488-831d-1187f1227138">

Reply:
Fixed!